cfme/intelligence/reports/menus.py

```diff
@@ -180,6 +194,43 @@ def manage_folder(self, group, folder=None):
             view.manager.commit()
             view.save_button.click()
 
+    def move_reports(self, group, folder, subfolder, reports):
+        """ Moves a list of reports to a given menu
+        Args:
+            group: User group
+            folder: Parent of the subfolder where reports are to be moved.
+            subfolder: Subfolder under which the reports are to be moved.
+            reports: List of reports that are to be moved.
+        """
+        if not isinstance(reports, list):
+            reports = [reports]
+
+        cancel_view = ''
+
+        with self.manage_subfolder(group, folder, subfolder) as selected_menu:
+            selected_options = selected_menu.parent_view.report_select.all_options
+            for report in reports:
+                if report in selected_options:
+                    # If the report is already present, we stay on the same page.
+                    cancel_view = self.create_view(EditReportMenusView)
+
+            # fill method replaces all the options in all_options with the value passed as argument
+            # We do not want to replace any value, we just want to move the new reports to a given
+            # menu. This is a work-around for that purpose.
+            reports.extend(selected_options)
+            selected_menu.parent_view.report_select.fill(reports)
+
+        view = self.create_view(AllReportMenusView)
```

Comment:
Please move this above L232. even I don't think you need to create this `AllReportMenusView` you are using only `flash` which is coming from Base view but its good practice that creates a spacific view :+1: